utilities/blob_db/blob_db_impl.cc

```diff
@@ -919,7 +920,7 @@ Status BlobDBImpl::Write(const WriteOptions& opts, WriteBatch* updates) {
       Writer::ConstructBlobHeader(&headerbuf, key, value, expiration, -1);
 
       if (previous_put) {
-        impl->AppendSN(last_file, -1);
+        impl->AppendSN(last_file, 0 /*sequence number*/);
```

Comment:
@anirbanr-fb I'm using 0 instead of -1 for dummy sequence number here, since `SequenceNumber` is an unsigned integer. Will GC be fine with it?